less/plugins/adapt-contrib-graphic/graphic.less

```diff
@@ -0,0 +1,7 @@
+.graphic {
+  &__image-container {
+    .img-zoom();
+    // .img-zoom(@duration: .5s;);
```

Comment:
Should these be deleted?

Reply:
Agree. Will move these to _animations.less file.